include/aspect/geometry_model/box.h

```diff
@@ -64,6 +64,11 @@ namespace aspect
          */
         void create_coarse_mesh (parallel::distributed::Triangulation<dim> &coarse_grid) const override;
 
+        /**
+         * Function to store data on current surface topography.
+         */
+        void update_surface () override;
```

Comment:
Like @gassmoeller already said, name it `update()`. There may in fact already be an `update()` function declared in the base class.

Reply:
Renamed to update.

Reply:
Renamed.